packages/core/src/components/alert/alert.scss

```diff
@@ -1,31 +1,28 @@
-@import "~@dso-toolkit/sources/src/styling/variables";
-
-@import "~@dso-toolkit/sources/src/utilities";
-@import "~@dso-toolkit/sources/src/components/sr-only/sr-only.mixins"; // stylelint-disable-line scss/at-import-partial-extension -- extension is needed because of filesystem based namespacing
-
-@import "~@dso-toolkit/sources/src/components/alert/alert";
+@use "~@dso-toolkit/sources/src/utilities";
+@use "~@dso-toolkit/sources/src/variables/units";
+@use "~@dso-toolkit/sources/src/components/alert/alert.mixins" as shared-alert-mixins;
```

Comment:
We moeten het toch nog ff over die `shared-` prefix hebben. Ik had niet helemaal helder dat dit te impact zou zijn.

Reply:
Zoals besproken; alles uit `sources` is leading en krijgt geen prefix. Wanneer een implementatie zijn eigen variabelen of mixins wil invliegen, krijgen die wel een prefix, bijv. `core-alert-variables`